sdk/servicebus/azservicebus/receiver.go

```diff
@@ -14,11 +24,306 @@ const (
 	SubQueueTransfer   = "transferDeadLetter"
 )
 
-type ServiceBusReceiver struct{}
+type Receiver struct {
```

Comment:
Doc comment.

Reply:
Many a doc comment has been added in my most recent push.

Also, since golint was so helpful I also fixed some other small things, like unnecessary else'ing, etc..